searx/engines/mysql_server.py

```diff
@@ -0,0 +1,57 @@
+# SPDX-License-Identifier: AGPL-3.0-or-later
+# lint: pylint
+# pylint: disable=missing-function-docstring
+"""MySQL database (offline)
+
+"""
+
+# import error is ignored because the admin has to install mysql manually to use
+# the engine
+import mysql.connector  # pylint: disable=import-error
+
+engine_type = 'offline'
+auth_plugin = 'caching_sha2_password'
+host = "127.0.0.1"
+database = ""
+username = ""
+password = ""
+query_str = ""
+limit = 10
+paging = True
+result_template = 'key-value.html'
+_connection = None
```

Comment:
This will raise an exception when the engine is loaded:
https://github.com/searxng/searxng/blob/da111f7fec1c377696ed33a3c97cb5665bb01ab9/searx/engines/__init__.py#L102-L104